docs/DeveloperGuide.md

```diff
@@ -13,6 +13,7 @@ title: Developer Guide
    - [Storage component](#storage-component)
    - [Common classes](#common-classes)
 5. [Implementation](#implementation)
+   - [[Proposed] View student/lesson feature](#view-studentlesson-feature)
```

Comment:
The `Proposed` tag here should be removed

Reply:
Right yes , forgot to remove it for the content page. Thx!